flutter/test/sentry_flutter_test.dart

```diff
@@ -1,22 +1,41 @@
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:package_info/package_info.dart';
 import 'package:sentry/sentry.dart';
-import 'package:sentry_flutter/default_integrations.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'sentry_flutter_util_test.dart';
 
 void main() {
-  const MethodChannel channel = MethodChannel('sentry_flutter');
+  const MethodChannel _channel = MethodChannel('sentry_flutter');
```

Comment:
why did you renamed it ? it's not a field, it should not have to be "private"

Reply:
just for the sake of encapsulation, it was not really needed